mantle/platform/qemu.go

```diff
@@ -415,13 +428,13 @@ func (builder *QemuBuilder) setupNetworking() error {
 // Mount9p sets up a mount point from the host to guest.  To be replaced
 // with https://virtio-fs.gitlab.io/ once it lands everywhere.
 func (builder *QemuBuilder) Mount9p(source, destHint string, readonly bool) {
-	builder.fs9pId += 1
+	builder.fs9pID++
```

Comment:
Huh, does the linter really prefer `++` over `+= 1`?

Reply:
Yep, https://github.com/golang/lint/blob/738671d3881b9731cc63024d5d88cf28db875626/lint.go#L1216-L1217.